src/more-infos/more-info-climate.html

```diff
@@ -46,22 +47,51 @@
         width: 100%;
       }
 
+      paper-range-slider {
+        width: 198px;
```

Comment:
paper-slider has a container for styling the size of the slider and paper-range-slider does not. I calculated the difference of the paper-slider and the paper-range-slider. Please let me know if these calculations are off.